README.md

```diff
@@ -14,7 +14,7 @@
 <br/><br/>
 
 ### Current State
-- <a href="https://www.figma.com/file/Ib05mHSvinzf9NNqIqftLR/Maakaf?type=design&node-id=51-508&mode=design&t=ldgKuFRDnEsudZvv-0">Figma Characterization and design</a> is to be complete soon (Special thanks to our UI/UX designer - Efrat Sharon!)
+- <a href="https://www.figma.com/file/qV5UBunxLQhqm41FfgRkmH/Maakaf?type=design&node-id=0-1&mode=design">Figma Characterization and design</a> is to be complete soon (Special thanks to our UI/UX designer - Efrat Sharon!)
```

Comment:
Try to open the link in incognito.
I'm afraid that for now the figma is not public.